WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -1488,11 +1497,9 @@ local methods = {
   end,
   ["HasUpdate"] = function(self)
     -- return hasUpdate, skipVersion, updateData, key
-    if WeakAurasCompanion
-    and self.data.uid
-    and not self.data.ignoreWagoUpdate
-    then
-      local slug = WeakAurasCompanion.uids[self.data.uid]
+    if not WeakAurasCompanion or self.data.ignoreWagoUpdate then return end
+    if self.data.uid then
+      local slug = WeakAurasCompanion.uids[self.data.uid] or WeakAurasCompanion.ids[self.data.id]
```

Comment:
WeakAurasCompanion.ids is a fallback if aura doesn't have a uid, as it it will never be used, it should be
`local slug = self.data.uid and WeakAurasCompanion.uids[self.data.uid] or WeakAurasCompanion.ids[self.data.id]`